### PR TITLE
Improve accessibility in the demo app

### PR DESCRIPTION
**Pull request by MGaetan89**
# Pull request

## Description

Improve accessibility in the demo app itself. These are mostly navigation-related changes. The player will be handled separately.

## Changes made

- Add language information to `DemoItem`s so TalkBack can use the proper voice when reading information.
- Add collection info on lists.
- Add some missing content descriptions.

## Checklist

- [ ] APIs have been properly documented (if relevant).
- [ ] The documentation has been updated (if relevant).
- [ ] New unit tests have been written (if relevant).
- [x] The demo has been updated (if relevant).

### CODE REVIEW

File: pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/settings/AppSettingsView.kt
Comment (StaehliJ) [inline pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/settings/AppSettingsView.kt:1]: The selected one should be the one focus by default and the selected one should be announced by TalkBack

**Review by StaehliJ — COMMENTED** [review]
Some little adjustement maybe, Shouldn't we not focus automatically the top when user choose an item in the bottom navigation bar ?

**Comment by MGaetan89** [comment]
> Some little adjustement maybe, Shouldn't we not focus automatically the top when user choose an item in the bottom navigation bar ?

I've checked in multiple Google apps, and the behavior is always the same: when we select a tab in the bottom navigation, the focus remains there.

File: pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/settings/AppSettingsView.kt
Comment (MGaetan89) [reply pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/settings/AppSettingsView.kt:1]: Not sure how to tell TalkBack to start with a specific item, but I've updated the dropdowns throughout the app so the selected item is clearly indicated.

**Review by StaehliJ — APPROVED** [review]

**Pull request merged**